### DISCUSSION TITLE
Would it be possible for drag drop plugin to drop jquery-ui ?

### DISCUSSION THREAD

**Discussion by giannik** [orchidjs/tom-select#322] [Q&A]
Im currently evaluating drag drop plugin for re-ordering selected items as shown here :

https://tom-select.js.org/plugins/drag-drop/

Would it be possible to drop jquery-ui and use a more lightwight library like sortable js  for the reordering of selected items ?
Jquery-ui is quite a big file and Im looking to make it more lightweight in terms of file size.
Thanks

**Comment by artur79** [comment]
that would be great

**Comment by ghost** [comment]
To replace it with vanilla js is simple, but the event from the parent component blocks it.
With additional libs like sortablejs it would be easy.
View https://github.com/orchidjs/tom-select/discussions/576

**Reply by ghost** [reply]
I added a hacky solution to my thread: https://github.com/orchidjs/tom-select/discussions/576
